spec/requests/foods_controller_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe 'Foods', type: :request do
+  context 'GET /index' do
+    before(:each) do
+      get '/foods'
+    end
+    it 'Foods index is successful' do
+      expect(response).to have_http_status(:ok)
+    end
+    it 'Renders correct template' do
+      expect(response).to render_template(:index)
+    end
+    it 'Does not render wrong template' do
+      expect(response).to_not render_template(:show)
+    end
+    it 'Response body includes correct placeholder text' do
+      expect(response.body).to include('LIST OF FOODS')
+    end
```

Comment:
- [ ] The Highlighted tests are failing. Please try to make them pass 👍 
![image](https://user-images.githubusercontent.com/46839548/202441428-759c1b2c-2683-492f-b238-5ab3e2754d6a.png)